core_lib/structure/layer.cpp

```diff
@@ -474,7 +474,7 @@ void Layer::mouseRelease( QMouseEvent* event, int frameNumber )
 
 void Layer::editProperties()
 {
-    QRegExp regex("([\uFFEF-\uFFFF])+");
+    QRegExp regex("([\\xFFEF-\\xFFFF])+");
```

Comment:
Have you tested this? as far as I know when you use \x and unicode characters, then you have to encapsulate them in braces.  So it should be [\x{FFEF}-\x{FFFF}] instead.